### PR TITLE
update language identifiers

### LINKED ISSUE #1737: Replace language identifier to use the supported ones for OPS
We need to add the list of supported language to our [style guide](https://github.com/dotnet/docs/blob/master/styleguide/template.md#inline-code-blocks-with-language-identifier) and replace the alternative ones currently in use in our repo to the officially supported ones, since new features will be done based on those.

For example, replace \`\`\`cs with \`\`\`csharp, \`\`\`fs with \`\`\`fsharp, etc.

The list of supported identifiers can be found in this internal site:
https://review.docs.microsoft.com/en-us/csi/msdocs/code?branch=master

/cc @tompratt-AQ

### PR DESCRIPTION
**Pull request by tompratt-AQ**
# Title
Replace language identifiers to use the supported ones for OPS (#1737)

## Summary
Changed the following language identifiers to match our [internal list of supported identifiers](https://review.docs.microsoft.com/en-us/csi/msdocs/code?branch=master
):
```cs``` TO ```csharp```, ```fs``` TO ```fsharp```, ```PHP``` to ```php```

Fixes #1737 

## Suggested Reviewers
@mairaw

### CODE REVIEW

**Comment by tompratt-AQ** [comment]
Resolved merge conflicts.

**Review by mairaw — CHANGES_REQUESTED** [review]
This looks great @tompratt-AQ! The only thing missing is to update the topic (https://github.com/dotnet/docs/blob/master/styleguide/template.md#inline-code-blocks-with-language-identifier) to have the list of supported identifiers instead of linking to https://github.com/jmm/gfm-lang-ids/wiki/GitHub-Flavored-Markdown-(GFM)-language-IDs. Can you please add that to this PR? Thanks.

File: docs/core/testing/unit-testing-with-dotnet-test.md [hunk 1]
```diff
@@ -151,8 +151,10 @@ Instead of creating new tests, leverage these two attributes to create a single
 
 Run `dotnet test`, and two of these tests fail. To make all of the tests pass, change the `if` clause at the beginning of the method:
 
+
 ```csharp
 if(candidate < 2)
```
Comment (mairaw) [inline docs/core/testing/unit-testing-with-dotnet-test.md]: for the merge conflicts, you should add the identifier csharp but keep the change that @GuardRex made to the topic: 
`if (candidate < 2)` (note he added a space after the if). and also remove the extra lines added.

File: docs/core/testing/unit-testing-with-mstest.md [hunk 2]
```diff
@@ -153,8 +153,10 @@ Instead of creating new tests, leverage these two attributes to create a single
 
 Run `dotnet test`, and two of these tests fail. To make all of the tests pass, change the `if` clause at the beginning of the method:
 
+
 ```csharp
 if(candidate < 2)
```
Comment (mairaw) [inline docs/core/testing/unit-testing-with-mstest.md]: same here.

**Comment by tompratt-AQ** [comment]
Updated the style guide link to point to the list of Supported Identifiers and fixed the merge conflicts.

File: styleguide/template.md [hunk 3]
```diff
@@ -254,7 +254,7 @@ runtime errors, you can use inline code blocks.
 
 ### Inline code blocks with language identifier
 
-Use three backticks (\`\`\`) + a language ID to apply language-specific color coding to a code block. Here is the entire list of [GFM language IDs](https://github.com/jmm/gfm-lang-ids/wiki/GitHub-Flavored-Markdown-(GFM)-language-IDs).
+Use three backticks (\`\`\`) + a language ID to apply language-specific color coding to a code block. Here is the list of [Supported Languages](https://review.docs.microsoft.com/en-us/csi/msdocs/code?branch=master#supported-languages).
```
Comment (mairaw) [inline styleguide/template.md]: customers won't have access to the list. you need to add the values from that topic here.

**Review by mairaw — CHANGES_REQUESTED** [review]
Left a comment. Plus something seems to have gone wrong with the rebase since there are some extra commits from other PRs showing up. I might need to help you to clean this up. Let me know.

**Pull request closed without merging**